rpc/jsonrpc/types/chainsvrresults.go

```diff
@@ -62,6 +62,7 @@ type GetAddedNodeInfoResult struct {
 // hex-encoded string.  Contains Decred additions.
 type GetBlockVerboseResult struct {
 	Hash          string        `json:"hash"`
+	PoWHash       string        `json:"powhash"`
```

Comment:
should probably be omitempty so it's not included in blocks prior to activation

Reply:
Makes it simpler for consumers to have it, but it's is just baggage prior to activation.  I'm fine either way personally.

Reply:
oh i misread. it's duplicating the block hash in these blocks.